src/metabase/driver/postgres.clj

```diff
@@ -161,6 +161,10 @@
      :visible-if   {"ssl-use-client-auth" true}}
     driver.common/ssh-tunnel-preferences
     driver.common/advanced-options-start
+    {:name         "json-unfolding"
+     :display-name (trs "Unfold JSON Columns")
+     :type         :boolean
+     :default      true}
```

Comment:
You'll need to add ` :visible-if  {"advanced-options" true}` so that the option's visibility is toggled correctly by on the `Show advanced options` button.

Also, should we have a description for this option?

Reply:
sgtm and we should prolly have a description